docs/blog/2023-Q2-community-board.rst

```diff
@@ -0,0 +1,58 @@
+.. post:: May 16, 2023
+   :tags: community board, news, report
+   :author: Mikey Ariel
+
+Write the Docs Community Board Quarterly Update Q2-2023
+=========================================================
+
+Greetings, documentarians! Mikey Ariel here, chairperson of the WTD community board.
+
+Last quarter we published our first `quarterly update blog post <https://www.writethedocs.org/blog/2023-Q1-community-board/>`_, where we introduced the team and gave an overview of how the board will aim to support and evolve the community. 
+
+Q2 quarterly meeting highlights
+-------------------------------
+
+Our second meeting focused on further refining our meta-work, such as where the community board tracks work and how we communicate with each other and with the community. 
+
+Historically, WTD used a Kanban board to track ideas for community initiatives and cross-functional tasks, such as infrastructure maintenance and meetup support. 
+Between our Q1 meeting and the Q2 meeting, we have spent some time reviewing and organizing this Kanban board, with the intention of landing on a "clean backlog" that will help us prioritize work and help the team leads to track their tasks, share resources, and ask for help. 
+
+During the meeting we looked at the main areas of the community that had active tasks in the backlog, and agreed to take the information back to our respective teams, as well as consider them as team leads for further triage in the upcoming months. 
+
+Another topic of discussion was the the #bipoc Slack channel and BIPOC stories in the newsletter. 
+Recently, it has become apparent that the community engagement was lower in those spaces, so in order to prevent burnout of the coordinator we recommended to take a breather from actively sourcing content. 
```

Comment:
```suggestion
Recently, it has become apparent that the community engagement was lower in those spaces, so in order to prevent burnout of the coordinator the newsletter and Slack team leads suggested to take a breather from actively sourcing content. 
```